xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -222,6 +222,7 @@ class CRenderManager
 
   struct SPresent
   {
+    SPresent();
```

Comment:
Are you talking about unit tests or real runtime tests? CRenderManager::Render should not be called if nothing valid is in the render buffers.
If a SPresent structure is used that was not filled with valid data, something has gone wrong before.